docker-compose.yml

```diff
@@ -11,6 +11,33 @@ services:
     volumes:
       - pgdata:/var/lib/postgresql/data
     restart: unless-stopped
+    networks:
+      - backend
+
+  redis:
+    image: redis:latest
+    expose:
+      - 6379
+    ports:
+      - "6379:6379"
```

Comment:
Deze is alleen nodig als je er vanaf buiten (=je laptop) tegenaan wil kletsen. Voor gebruik binnen de dockers zelf is het niet nodig.